src/main/java/org/jvnet/hudson/test/junit/jupiter/EnableJenkins.java

```diff
@@ -0,0 +1,63 @@
+package org.jvnet.hudson.test.junit.jupiter;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+/**
+ * JUnit 5 meta annotation providing {@link org.jvnet.hudson.test.JenkinsRule JenkinsRule} integration.
+ *
+ * <p>Test methods using the rule extension need to accept it by {@link org.jvnet.hudson.test.JenkinsRule JenkinsRule} parameter;
+ * each test case gets a new rule object.
```

Comment:
> Assuming this is correct

I do not know if this is correct or not, so I can neither accept nor decline this suggestion.

Reply:
@offa As the original author of this code, are you able to address Daniel's concern?

Reply:
Yes, a test gets an rule object only if it's annotated and accepts an appropriate parameter ([doc](https://junit.org/junit5/docs/current/user-guide/#writing-tests-dependency-injection)).

> Ideally this would cause an error for easier diagnosability.

An error or warning would be really annoying. Tests not needing the rule would have to declare unused parameters (eg. an annotated test suite with some tests, not using the rule).

Reply:
Thanks @offa. Based no your comments I accepted Daniel's suggestion.